avm/res/db-for-my-sql/flexible-server/configuration/main.bicep

```diff
@@ -0,0 +1,36 @@
+metadata name = 'DBforMySQL Flexible Server Configurations'
+metadata description = 'This module deploys a DBforMySQL Flexible Server Configuration.'
+
+@description('Required. The name of the configuration.')
+param name string
+
+@description('Conditional. The name of the parent MySQL flexible server. Required if the template is used in a standalone deployment.')
+param flexibleServerName string
+
+@description('Optional. Source of the configuration.')
+param source string?
```

Comment:
```suggestion
@allowed([
  'system-default',
  'user-override'
])
param source string?
```

Reply:
Note: This will require a regeneration of the json & readmes.
In general, I recommend to regenerate both for all files once all changes are implemented (if any)